src/components/switch/switch.component.tsx

```diff
@@ -214,16 +214,23 @@ export const Switch = React.forwardRef(
       ...rest,
     };
 
+    const applyValidation = error || warning;
+
     return (
       <>
         {validationRedesignOptIn ? (
           <StyledSwitch {...switchStylePropsForNewValidation}>
             <Label>
-              {label}
-              {labelHelp && <StyledHintText>{labelHelp}</StyledHintText>}
+              <Box
+                ml={applyValidation ? "0px" : "2px"}
+                mb={labelHelp ? "0px" : "8px"}
+              >
+                {label}
+                {labelHelp && <StyledHintText>{labelHelp}</StyledHintText>}
+              </Box>
               <Box position="relative">
                 <ValidationMessage error={error} warning={warning} />
-                {(error || warning) && (
+                {applyValidation && (
                   <ErrorBorder warning={!!(!error && warning)} />
```

Comment:
comment: I realise this isn't part of this PR but there's no need to coerce here it's already a boolean and I can't un-see it

```suggestion
                  <ErrorBorder warning={!error && warning} />
```

Reply:
Yeah, I remember seeing this originally and wondered why it was done this was. I just followed the pattern that was already in place originally. I'll happily get this changed though 😆

Reply:
Haven't made this change as per our private conversation btw.

Reply:
No worries, looks like TS can't infer the type correctly here nvm